binder/requirements.txt

```
@@ -0,0 +1,2 @@
+-r ../dev_requirements.txt
+.
```

```suggestion
.[development]
```

This is much faster (pip resolves dependencies faster) and corresponds to the command that we use in GitHub for CI.

---

Nice!